### DISCUSSION TITLE
Loop a video

### DISCUSSION THREAD

**Discussion by Carunga** [mkdocs/mkdocs#3120] [Q&A] [answered]
Hello everyone,
I want to embed short videos that should play in a loop until stopped.  Basically what could be done with
```
 <video loop src="movie.mp4">  video </video> 
```
What would be a good way to get this done? The video plugin does not support looping I believe. HTML attributes cannot be changed using extra css (I think). Using extra javascript did not work either (which might be due to my non existing scripting skills). Using a hook to change the mkdocs generated <video ...> tag to <video loop ...> would do the trick for me - but  I could not figure out how to get this done.

Thanks for any pointers in the right direction!

**Comment by oprypin** [comment]
> could be done with
> ```html
>  <video loop src="movie.mp4">  video </video> 
> ```

Soo just put exactly this HTML snippet into the Markdown file.

**Reply by oprypin** [reply]
Also wrote a comment to discourage the use of that plugin https://github.com/mkdocs/mkdocs/issues/243#issuecomment-1417499597

Let me know how you found the plugin and I can write this there too 😅

**Comment by Carunga** [comment]
@oprypin Thanks for your answer. The HTML-Tag is a obvious suggestion. Tried this in the very beginning and forgot about it again ;)

My markdown-editor of choice obsidian.md does not display videos inserted with video-html-tag  and I do not understand why. Also there will  be a lot of videos so using the ![video](video.mp4] would be easier. That's why I started using the plugin in the first place.

And I do have some other issues with the video tag, but this might just be due to my weird file structure I realized now testing again. I have to look into this some more but wanted to thank you for your suggestions timely.

**Comment by Carunga** [comment, marked as answer]
@soulless-viewer fixed the issue and added support for the loop attribute to this plugin mkdocs-video.
